src/index.ts

```diff
@@ -146,6 +145,10 @@ export class MongoLogWriter extends Writable {
       EJSON.stringify(fullInfo.attr);
     } catch {
       try {
+        // This package may be running in a web environment
+        // where v8 is not available.
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const v8 = require('v8');
```

Comment:
Would it be better to try to import this once in the constructor instead of dynamically importing every time `write` is run? or will it be insignificant?

Reply:
Yeah, it might be slightly better to cache this  -- but I wouldn't expect the difference to be big. :)